transaction-status/src/lib.rs

```diff
@@ -132,49 +132,54 @@ pub struct TransactionWithStatusMeta {
 pub enum TransactionEncoding {
     Binary,
     Json,
+    JsonParsed, // same as Json in 1.1
```

Comment:
I snuck in a stub `JsonParsed` to ensure the enum values will line up between v1.3/v1.2 and v1.1